gravitee-apim-console-webui/src/management/settings/shared-policy-groups/shared-policy-groups.component.html

```diff
@@ -89,7 +89,10 @@
           <th mat-header-cell *matHeaderCellDef></th>
           <td mat-cell *matCellDef="let element">
             <div class="actions_btn">
-              @if (isReadOnly) {
+              @if (isReadOnly || element.isReadOnly) {
+                <div class="shared-policy-groups__origin-kubernetes">
```

Comment:
hum maybe it's not always kubernetes if the SPG are in ReadOnly mode. no ?

Reply:
OK as we show a kubernetes icon here, i'll change the flag to isKubernetesOrigin :)